Main.java

```diff
@@ -1,15 +1,34 @@
-/**
- *
- * @author 
- */
-public class Main {
+import java.util.Scanner;
+/**
+ * This program will put 2 entered integers in ascending order
+ * @Chad 
```

Comment:
@author is actually a special tag belonging to something called javadoc. So this line should be 
@author Chad